repo/packages/J/jenkins/0/marathon.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "jenkins",
-  "instances": 1,
-  "mem": 1500,
+  "instances": 1.5,
+  "mem": 2048,
   "env": {
     "JAVA_OPTS": "-Xmx300m"
```

Comment:
Don't you need to change this too? Doesn't look like you are using it in `cmd`. Might as well remove it.